app/models/drug_stock.rb

```diff
@@ -32,4 +46,10 @@ def self.latest_for_facilities_cte(facilities, for_end_of_month)
     from(latest_for_facilities(facilities, for_end_of_month), table_name)
       .includes(:protocol_drug)
   end
```

Comment:
If we can manage to drop the `facility` from drug stocks, we could remove these `latest_for_facilities*` methods too, right?

Reply:
Yes 👍🏼